btree/btree.c

```diff
@@ -2839,18 +2912,35 @@ static void generic_move(struct nd *src, struct nd *tgt,
 	node_fix(src, tx);
 	node_seq_cnt_update(tgt);
 	node_fix(tgt, tx);
+}
 
-	/**
-	 * ToDo: We need to capture the changes occuring in the "src" node
-	 * using m0_be_tx_capture().
-	 * Only the modified memory from the node needs to be updated.
-	 */
+static bool ff_capture(struct slot *slot, struct m0_be_tx *tx)
+{
+	struct ff_head   *h     = ff_data(slot->s_node);
+	int               rsize  = h->ff_ksize + h->ff_vsize;
+	void             *start = ff_key(slot->s_node, slot->s_idx);
+	struct m0_be_seg *seg   = slot->s_node->n_tree->t_seg;
+	m0_bcount_t       hsize = sizeof(*h) - sizeof(h->ff_opaque);
 
+	rsize = rsize;
 	/**
```

Comment:
Is this a typo?

Reply:
No this is needed to keep the compiler happy. When we remove the dummy macro then we do not need this line but till then I will need to keep it.
Let me try something else in the macro to avoid this statement.

Reply:
ok.